client/my-sites/checkout/composite-checkout/record-analytics.js

```diff
@@ -351,15 +398,15 @@ export default function createAnalyticsEventHandler( reduxDispatch ) {
 	};
 }
 
-function logStashEventAction( type, message, additionalData = {} ) {
+function logStashEventAction( type, payload, additionalData = {}, message ) {
 	return logToLogstash( {
 		feature: 'calypso_client',
-		message: 'composite checkout load error',
+		message: message ?? 'composite checkout load error',
 		severity: config( 'env_id' ) === 'production' ? 'error' : 'debug',
 		extra: {
 			env: config( 'env_id' ),
 			type,
-			message,
+			payload,
```

Comment:
Just for the sake of keeping our logs the same, which arguably isn't that important, could we keep the same key on this property?
```suggestion
			message: payload,
```